### PR TITLE
palindrome verfication

### PR DESCRIPTION
**Pull request by Abbas-Abdulrab**

### CODE REVIEW

**Review by Rudainasaleh — APPROVED** [review]

**Review by sarah-alshukaily — DISMISSED** [review]

File: examples/abbas_work_area/palindrome.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+user = input("Enter: ").strip().replace(" ", "").lower()
```
Comment (CodelineAtyab) [inline examples/abbas_work_area/palindrome.py:1]: Please consider using a meaningful variable name like user_input or something else

File: examples/abbas_work_area/palindrome.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+punc = []
```
Comment (CodelineAtyab) [inline examples/abbas_work_area/palindrome.py:2]: Please consider using a meaningful variable name like list_of_punctuation_characters or something like this

File: examples/abbas_work_area/palindrome.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+for curr in user:
+    if curr not in [",", "-", "."]:
+        punc.append(curr)
+
+remove = "".join(punc)
```
Comment (CodelineAtyab) [inline examples/abbas_work_area/palindrome.py:8]: string_without_punctuation or something like this

File: examples/abbas_work_area/palindrome.py [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+punc1 = punc[::-1]
```
Comment (CodelineAtyab) [inline examples/abbas_work_area/palindrome.py:10]: Please add comments to make it clear, what is being done here

File: examples/abbas_work_area/palindrome.py [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+if punc == punc1:
+    print("True")
```
Comment (CodelineAtyab) [inline examples/abbas_work_area/palindrome.py:13]: Lets put this logic in a function and return True OR False from that function

**Review by CodelineAtyab — CHANGES_REQUESTED** [review]
Please respond to comments

**Review by ziyadabd — APPROVED** [review]

**Review by aaziz9 — APPROVED** [review]

**Review by Sughiya-AlSaid — APPROVED** [review]

**Review by CodelineAtyab — APPROVED** [review]
Nice!

**Pull request merged**